DataFormats/TotemReco/interface/TotemT2Digi.h

```diff
@@ -12,12 +12,13 @@
 class TotemT2Digi {
 public:
   TotemT2Digi() = default;
-  TotemT2Digi(unsigned short id, unsigned char marker, unsigned short le, unsigned short te);
+  TotemT2Digi(unsigned short id, unsigned char marker, unsigned short le, unsigned short te, unsigned char status);
 
   void setLeadingEdge(unsigned short le) { lead_edge_ = le; }
   unsigned short leadingEdge() const { return lead_edge_; }
   void setTrailingEdge(unsigned short te) { trail_edge_ = te; }
   unsigned short trailingEdge() const { return trail_edge_; }
+  unsigned char getStatus() const { return status_ & 0xF; }
```

Comment:
This is very minor, but if you need to implement any other modification to the PR, the name of this method should be preferably `status()`, without the "get" (see item 11 in http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1)